src/swish/io.ss

```diff
@@ -54,8 +58,12 @@
    open-file-to-write
    open-utf8-bytevector
    osi-port-count
+   osi-port-create-time
+   osi-port-name
+   osi-port?
```

Comment:
Would it make sense to provide an `osi-port-closed?` predicate? I'm not convinced it's worth doing. The obvious implementation would tell us only about the status of `handle`, not the underlying `fd`.

Reply:
Yes, good idea.